plenum/server/monitor.py

```diff
@@ -493,6 +573,15 @@ def __init__(self, name: str, Delta: float, Lambda: float, Omega: float,
         self.latency_avg_strategy_cls = self.config.latency_averaging_strategy_class
         self.throughput_avg_strategy_cls = self.config.throughput_averaging_strategy_class
 
+        config = getConfig()
+        self.acc_monitor_enabled = config.ACC_MONITOR_ENABLED
+        self.acc_monitor = AccumulatingMonitorStrategy(
```

Comment:
Why is this monitor created unconditionally despite _acc_monitor_enabled_ may be False? And further working with this monitor is unconditional too, just _is_master_degraded()_ is called under condition.

Reply:
My bad. I was thinking about disabling visible effects (that's why condition is checked in _is_master_degraded_), but was confident that this strategy is lightweight enough so it could be processed anyways. In fact this helped to catch some internal bugs with existing integration tests. But yes, on the second thought I think this should be fixed.

Reply:
Yeah, even if this is lightweight it is unnecessary work that shouldn't be done in case of disabled ACC monitor.